pkg/apis/triggers/v1alpha1/trigger_binding_types.go

```diff
@@ -0,0 +1,74 @@
+package v1alpha1
+
+/*
+Copyright 2019 The Tekton Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import (
+	pipelinev1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Check that TriggerBinding may be validated and defaulted.
+//var _ apis.Validatable = (*TriggerBinding)(nil)
+
+// +k8s:deepcopy-gen=true
+type TriggerBindingSpec struct {
+	Templates []TemplateBinding `json:"templates,omitempty"`
```

Comment:
nit: can we please rename this to `TemplateBindings` with `json:"templatebindings,omitempty"`?

Reply:
ty, updated!